app/models/user.rb

```diff
@@ -151,9 +151,11 @@ def mhv_correlation_id
     identity.mhv_correlation_id || mpi_mhv_correlation_id
   end
 
-  def mhv_user_account
+  def mhv_user_account(break_cache: false)
+    @mhv_user_account = nil if break_cache
```

Comment:
Something I was thinking about, if the `MHVUserAccountsController` is calling the creator directly, maybe the User model doesn't have to deal with breaking the cache at all? Cause `@mhv_user_account` will pull the latest from `MHV::UserAccount::Creator` whenever a request is made, and it'll stay consistent throughout the request. But since this isn't a cached attribute on the User model redis we don't have to worry about it persisting beyond a single request

Reply:
Yeah that makes sense. Could there ever be a case where someone performs an action and then wants a fresh copy on the user? i guess we can add it in the future if we need it